### ISSUE TITLE
gcc-2_95-branch updates

### ISSUE THREAD

**Issue by sezero** [cahirwpz/amigaos-gcc-2.95.3#3]
I made a fork and applied gcc-2_95-branch changes (gcc-2.95.4-prerelease),
and a few other minor updates from PLD, debian, etc:
https://github.com/sezero/amigaos-gcc-2.95.3/tree/2_95_4

If you are interested, in all or some of the changes, either take them from there
or tell me to create a PR here.

**Comment by cahirwpz** [comment]
I agree those patches could, and probably should, be merged. But until full test suite is run on **gcc**, thanks to @cnvogelg's  fixed`vamos`, I'm not going to modify the sources.

**Comment by sezero** [comment]
Ok. The branch is there, whenever you want them.

**Comment by jens-maus** [comment]
Great to see these changes being proposed to being merged. I also agree that we should merge in the latest 2.95.4 changes as @sezero propose here. I know that @cahirwpz likes to have a full test suite being in place to verify the effect of changes. However, I would still vote for not waiting until you have your test suite completely ready (which could easily take months, I guess) but use the opportunity to merge these changes and let developer like me (with a lot of Amiga related projects) somehow beta test them because if there would be a negative effect on large projects like MUI, AmiSSL, YAM, etc. we would immediately notice it, I guess.
